### DISCUSSION TITLE
How to know when a post is unhidden?

### DISCUSSION THREAD

**Discussion by fredstluka** [community/community#22381] [New to GitHub]
<p>How to know when a post is unhidden?</p>
<p>I joined the GitHub Support Community forum recently and posted 2 questions.  I immediately got 2 automated emails like:</p>
<blockquote>
<p>Our automated spam filter, <a href="https://akismet.com/" rel="nofollow noopener">Akismet</a>, has temporarily hidden <a href="https://github.community/t/how-to-forward-an-issue-to-myself-as-email/118224/1">your post</a> in <em>How to forward an issue to myself as email?</em> for review.</p>
<p>A staff member (link deleted, see below --Fred) will review your post soon, and it should appear shortly.</p>
<p>We apologize for the inconvenience.</p>
</blockquote>
<p>I clicked on the links and was unable to see my posts.</p>
<p>I assumed I’d get more automated emails when they were reviewed and approved.  In an ideal world, the staff member reviewing them would have time to also post a reply.</p>
<p>3 days later, I’ve received no further emails.  I clicked the links again, and can now see my posts.  But I don’t know if others can see them, and there have been no replies.</p>
<p>I went to the URLs of the posts with another browser, where I was not logged in to the GitHub Support Community forum.  So, that should be the same as anyone else trying to view the posts, I think.  I can see them then also.  Does that mean they are now fully visible?</p>
<p>Is anyone likely to see and respond to them?  Or are they likely to go unnoticed since they’re now 3 days old and have dropped further down any list that’s shown in date order?</p>
<p>I’m assuming that the hide/review process was only because I was a brand new member, and won’t be done to every post I make.  Perhaps I should repost the 2 questions, so they’ll appear more quickly at the top of any list of new posts?  Or is that considered rude?</p>
<p>BTW, I had to delete the links in the quoted email above because I got a message saying:</p>
<blockquote>
<p>Sorry, new users can only put 2 links in a post.</p>
</blockquote>
<p>Thanks,<br>
–Fred</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<aside class="quote no-group" data-username="fredstluka" data-post="1" data-topic="118701">
<div class="title">
<div class="quote-controls"></div>
 fredstluka:</div>
<blockquote>
<p>I went to the URLs of the posts with another browser, where I was not logged in to the GitHub Support Community forum. So, that should be the same as anyone else trying to view the posts, I think. I can see them then also. Does that mean they are now fully visible?</p>
</blockquote>
</aside>
<p>Correct. If you can see them when logged out, they are visible.</p>
<aside class="quote no-group" data-username="fredstluka" data-post="1" data-topic="118701">
<div class="title">
<div class="quote-controls"></div>
 fredstluka:</div>
<blockquote>
<p>Is anyone likely to see and respond to them? Or are they likely to go unnoticed since they’re now 3 days old and have dropped further down any list that’s shown in date order?</p>
</blockquote>
</aside>
<p>We go through flagged posts almost every day other than weekends. It is unlikely that a post will remain hidden more than a day unless it is a weekend when our volumes in the forum is lower. So your post might be a little lower in the viewing, but it shouldn’t drop that far.</p>
<p>We also have a team looking for posts that don’t have any responses on questions about GitHub and GitHub products. While we don’t get to every questions by any means, we are actively looking to make sure that questions don’t fall through the cracks. We are also still working on ways to improve this process.</p>
<aside class="quote no-group" data-username="fredstluka" data-post="1" data-topic="118701">
<div class="title">
<div class="quote-controls"></div>
 fredstluka:</div>
<blockquote>
<p>I’m assuming that the hide/review process was only because I was a brand new member, and won’t be done to every post I make.</p>
</blockquote>
</aside>
<p>That will have something to do with how the spam filter looks at your posts. But it may also be the content of your posts as well. Since I didn’t process these posts, I am unable to tell you definitively why your posts got flagged by the automated system.</p>
<aside class="quote no-group" data-username="fredstluka" data-post="1" data-topic="118701">
<div class="title">
<div class="quote-controls"></div>
 fredstluka:</div>
<blockquote>
<p>Perhaps I should repost the 2 questions, so they’ll appear more quickly at the top of any list of new posts? Or is that considered rude?</p>
</blockquote>
</aside>
<p>Please don’t repost questions. Creating duplicate topics and bumping your own posts goes against the community guidelines. However, I don’t want you to feel stuck in this situation either. If you would share links to your two posts without responses, I will take a look myself.</p>
<aside class="quote no-group" data-username="fredstluka" data-post="1" data-topic="118701">
<div class="title">
<div class="quote-controls"></div>
 fredstluka:</div>
<blockquote>
<p>I assumed I’d get more automated emails when they were reviewed and approved. In an ideal world, the staff member reviewing them would have time to also post a reply.</p>
</blockquote>
</aside>
<p>Honestly, I did not realize that emails weren’t sent out for when responses were unhidden. I made a similar assumption that you did and will ask internally about this to see if we can make that change if that isn’t already a feature.</p>
<p>I do agree that in an ideal world, staff resolving these spam flags would have time to respond as well. However, just to be transparent, we separate these tasks for a couple of reasons. Firstly, we want to get through the moderation queue as quickly as possible to allow us to unhide posts that shouldn’t be hidden and allow the community to see them quicker.</p>
<p>Answering questions can take some time, especially when switching from one type of cognitive task to another, so we try to minimize that sort of context switching. We also have different people who specialize in different questions because of the sheer size of the GitHub product makes it hard to be an expert in all areas. Because of this, the person that releases the flag on a post might not be the person capable of giving you a response.</p>
<hr>
<p>I appreciate you taking the time to write to us about your experience. I will write in to see what we can do about getting automated emails sent out when a post is released. I’m not sure what that will take to accomplish, but I think that’s a good idea to improve the user experience.</p>

**Comment by fredstluka** [comment]
<aside class="quote group-github-employees" data-username="that-pat" data-post="2" data-topic="118701">
<div class="title">
<div class="quote-controls"></div>
 that-pat:</div>
<blockquote>
<p>Correct. If you can see them when logged out, they are visible.</p>
</blockquote>
</aside>
<p>OK.  Thanks!</p>
<aside class="quote group-github-employees" data-username="that-pat" data-post="2" data-topic="118701">
<div class="title">
<div class="quote-controls"></div>
 that-pat:</div>
<blockquote>
<p>We go through flagged posts almost every day other than weekends. It is unlikely that a post will remain hidden more than a day unless it is a weekend when our volumes in the forum is lower. So your post might be a little lower in the viewing, but it shouldn’t drop that far.</p>
</blockquote>
</aside>
<p>OK.  Yeah, they only dropped down a few dozen entries.  I’m not sure how far that is effectively.  Depends on the process used by you and others who might see and answer my questions.  They HAVE dropped “below the fold” so to speak.</p>
<aside class="quote group-github-employees" data-username="that-pat" data-post="2" data-topic="118701">
<div class="title">
<div class="quote-controls"></div>
 that-pat:</div>
<blockquote>
<p>We also have a team looking for posts that don’t have any responses on questions about GitHub and GitHub products. While we don’t get to every questions by any means, we are actively looking to make sure that questions don’t fall through the cracks.</p>
</blockquote>
</aside>
<p>Perfect!  I’ve been a programmer for 35+ years, and often have to monitor a ticket system to support users.  So, I understand how hard it can be to keep up with the large volume of questions and the wide range of user expertise.</p>
<aside class="quote group-github-employees" data-username="that-pat" data-post="2" data-topic="118701">
<div class="title">
<div class="quote-controls"></div>
 that-pat:</div>
<blockquote>
<p>We are also still working on ways to improve this process.</p>
</blockquote>
</aside>
<p>Where’s the best place to post suggestions about this GitHub Community Support forum, as opposed to GitHub itself?  I posted this ticket to category:</p>
<ul>
<li>“How to use Git and GitHub”</li>
</ul>
<p>Later, I noticed there’s another category:</p>
<ul>
<li>“Welcome and Announcements”</li>
</ul>
<p>Its description says</p>
<ul>
<li>“…or ask questions about the GitHub Support Community itself”.</li>
</ul>
<p>Should I post suggestions there?  Or is there a separate bug/feature tracking system that I should use?  I have some ideas already and would be happy to post them.</p>
<p>As I said, I’ve used a LOT of version control systems over the years (VAX CMS, Unix sccs, Unix rcs, MS VSS, PVCS, CVS, and then finally Git).  And a dozen or more ticket tracking systems.</p>
<p>Here’s a tip I wrote a while back about the advantages of Git and some shell aliases I use for common Git commands:</p>
<ul>
<li>xhttp://bristle.com/Tips/Programming.htm#git</li>
</ul>
<p>(Delete the “x” from the "xhttp: above to use the link.  I got an error when I tried to post with a link to my site, saying “Sorry you cannot post a link to that host.”</p>
<aside class="quote group-github-employees" data-username="that-pat" data-post="2" data-topic="118701">
<div class="title">
<div class="quote-controls"></div>
 that-pat:</div>
<blockquote>
<p>That will have something to do with how the spam filter looks at your posts. But it may also be the content of your posts as well. Since I didn’t process these posts, I am unable to tell you definitively why your posts got flagged by the automated system.</p>
</blockquote>
</aside>
<p>OK.</p>
<aside class="quote group-github-employees" data-username="that-pat" data-post="2" data-topic="118701">
<div class="title">
<div class="quote-controls"></div>
 that-pat:</div>
<blockquote>
<p>Please don’t repost questions. Creating duplicate topics and bumping your own posts goes against the community guidelines.</p>
</blockquote>
</aside>
<p>Yeah, I thought it might.  That’s why I asked first.  Thanks!</p>
<aside class="quote group-github-employees" data-username="that-pat" data-post="2" data-topic="118701">
<div class="title">
<div class="quote-controls"></div>
 that-pat:</div>
<blockquote>
<p>However, I don’t want you to feel stuck in this situation either. If you would share links to your two posts without responses, I will take a look myself.</p>
</blockquote>
</aside>
<p>OK.  Here ya go:</p>
<ul>
<li><a href="https://github.community/t/how-to-get-notified-when-an-issue-is-edited/118216" class="inline-onebox">How to get notified when an issue is EDITED?</a></li>
<li><a href="https://github.community/t/how-to-forward-an-issue-to-myself-as-email/118224" class="inline-onebox">How to forward an issue to myself as email?</a></li>
</ul>
<aside class="quote group-github-employees" data-username="that-pat" data-post="2" data-topic="118701">
<div class="title">
<div class="quote-controls"></div>
 that-pat:</div>
<blockquote>
<p>Honestly, I did not realize that emails weren’t sent out for when responses were unhidden. I made a similar assumption that you did and will ask internally about this to see if we can make that change if that isn’t already a feature.</p>
</blockquote>
</aside>
<p>OK.  Thanks!  I’ll be glad to submit a ticket to your tracking system, if any.  Or post them to this forum as separate posts.  Then I’ll get email notifications when the feature is added, rejected, or commented on.  Let me know where to post it.</p>
<aside class="quote group-github-employees" data-username="that-pat" data-post="2" data-topic="118701">
<div class="title">
<div class="quote-controls"></div>
 that-pat:</div>
<blockquote>
<p>I do agree that in an ideal world, staff resolving these spam flags would have time to respond as well. However, just to be transparent, we separate these tasks for a couple of reasons. Firstly, we want to get through the moderation queue as quickly as possible to allow us to unhide posts that shouldn’t be hidden and allow the community to see them quicker.</p>
</blockquote>
</aside>
<p>I understand perfectly.  Been there, many times…</p>
<aside class="quote group-github-employees" data-username="that-pat" data-post="2" data-topic="118701">
<div class="title">
<div class="quote-controls"></div>
 that-pat:</div>
<blockquote>
<p>Answering questions can take some time, especially when switching from one type of cognitive task to another, so we try to minimize that sort of context switching.</p>
</blockquote>
</aside>
<p>Exactly!  I once warned a manager that context switches were expensive.  Especially for a programmer, who’s holding a bunch of things in his head all at once, and can’t let any of them slip away without writing them down to be dealt with later.</p>
<p>Things that get forgotten due to an interruption or other context switch may eventually show up as bugs.  Also, at the start of each new context, you have to load a bunch of things into your head to remember how it all works, before you can start making changes.</p>
<p>I estimated that each context switch cost the company about 4 hours of programmer time, perhaps much more.  See the articles at the “Holding a Program in Your Head” row of my links page:</p>
<ul>
<li>xhttp://bristle.com/~fred/<span class="hashtag">#holding_a_program_in_your_head</span>
</li>
</ul>
<p>(Delete the “x” from the "xhttp: above to use the link.  I got an error when I tried to post with a link to my site, saying “Sorry you cannot post a link to that host.”</p>
<p>With titles like:</p>
<ul>
<li>Why programmers work at night</li>
<li>Holding a program in your head</li>
<li>Programmer interrupted</li>
<li>etc.</li>
</ul>
<aside class="quote group-github-employees" data-username="that-pat" data-post="2" data-topic="118701">
<div class="title">
<div class="quote-controls"></div>
 that-pat:</div>
<blockquote>
<p>We also have different people who specialize in different questions because of the sheer size of the GitHub product makes it hard to be an expert in all areas. Because of this, the person that releases the flag on a post might not be the person capable of giving you a response.</p>
</blockquote>
</aside>
<p>Makes sense.</p>
<aside class="quote group-github-employees quote-modified" data-username="that-pat" data-post="2" data-topic="118701">
<div class="title">
<div class="quote-controls"></div>
 that-pat:</div>
<blockquote>
<p>I appreciate you taking the time to write to us about your experience. I will write in to see what we can do about getting automated emails sent out when a post is released. I’m not sure what that will take to accomplish, …</p>
</blockquote>
</aside>
<p>As a programmer, I’m guessing it won’t be too hard.  You already have a trigger that sends the “blocked” email when a new user posts something.  The automatic sending of “unblocked” email should be very similar.  Build it into the button or process that the reviewer clicks or uses to unblock the post.</p>
<p>But, of course, there may be more important new features on the wish list, and there are limited developer resources to do them all.</p>
<aside class="quote group-github-employees quote-modified" data-username="that-pat" data-post="2" data-topic="118701">
<div class="title">
<div class="quote-controls"></div>
 that-pat:</div>
<blockquote>
<p>… but I think that’s a good idea to improve the user experience.</p>
</blockquote>
</aside>
<p>Absolutely!  Make things as pleasant as possible for GitHub users, especially new users.  If they get frustrated, they’ll all go to Bitbucket or GitLab.</p>
<p>–Fred</p>

**Comment by ThirdEyeRose** [comment]
<aside class="quote no-group" data-username="fredstluka" data-post="3" data-topic="118701">
<div class="title">
<div class="quote-controls"></div>
 fredstluka:</div>
<blockquote>
<p>Where’s the best place to post suggestions about this GitHub Community Support forum, as opposed to GitHub itself?</p>
</blockquote>
</aside>
<p>Honestly, the best way to get our attention about something in the forum is by using the <strong>Feedback</strong> button on the right hand side. We don’t have a category that is dedicated to the forum itself at this time.</p>
<aside class="quote no-group" data-username="fredstluka" data-post="3" data-topic="118701">
<div class="title">
<div class="quote-controls"></div>
 fredstluka:</div>
<blockquote>
<p>Later, I noticed there’s another category:</p>
<ul>
<li>“Welcome and Announcements”</li>
</ul>
<p>Its description says</p>
<ul>
<li>“…or ask questions about the GitHub Support Community itself”.</li>
</ul>
</blockquote>
</aside>
<p>Welcome and Announcements would also work.</p>
<aside class="quote no-group quote-modified" data-username="fredstluka" data-post="3" data-topic="118701">
<div class="title">
<div class="quote-controls"></div>
 fredstluka:</div>
<blockquote>
<p>OK. Here ya go:</p>
<ul>
<li><a href="https://github.community/t/how-to-get-notified-when-an-issue-is-edited/118216">How to get notified when an issue is EDITED?</a></li>
<li><a href="https://github.community/t/how-to-forward-an-issue-to-myself-as-email/118224">How to forward an issue to myself as email?</a></li>
</ul>
</blockquote>
</aside>
<p>Thanks for sharing. I’ll take a look right now.</p>
<aside class="quote no-group quote-modified" data-username="fredstluka" data-post="3" data-topic="118701">
<div class="title">
<div class="quote-controls"></div>
 fredstluka:</div>
<blockquote>
<p>As a programmer, I’m guessing it won’t be too hard. You already have a trigger that sends the “blocked” email when a new user posts something. The automatic sending of “unblocked” email should be very similar. Build it into the button or process that the reviewer clicks or uses to unblock the post.</p>
<p>But, of course, there may be more important new features on the wish list, and there are limited developer resources to do them all.</p>
</blockquote>
</aside>
<p>Yeah, the technical issue probably isn’t that hard in of itself. However, we’re using a platform that is not home-brewed so beyond the specific technical limitations, we also are limited by the platform’s decisions and priorities. We’re talking with them now to find out what our options are.</p>

**Comment by mattmccutchen** [comment]
<p>I too have <a href="https://github.community/t/using-one-account-for-all-your-projects/10197/7">a post</a> that has been hidden by Akismet for almost 48 hours now; I’ll contact the moderators separately about that specific post.  But I have some general process points that I think are on topic for this thread:</p>
<aside class="quote no-group" data-username="that-pat" data-post="4" data-topic="118701">
<div class="title">
<div class="quote-controls"></div>
 that-pat:</div>
<blockquote>
<p>Honestly, the best way to get our attention about something in the forum is by using the <strong>Feedback</strong> button on the right hand side. We don’t have a category that is dedicated to the forum itself at this time.</p>
</blockquote>
</aside>
<p>I can’t find this button anywhere.  Here’s what I see:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/a/3a5032a1e7cc38b1125587cab20749939928d519.jpeg" data-download-href="/uploads/short-url/8jRw0rjx4dr30DxkYfnwfwkG6hP.jpeg?dl=1" title="" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/8885753/181070680-b5ec7846-d5c9-4ef8-8283-db9266268930.jpeg" alt="" data-base62-sha1="8jRw0rjx4dr30DxkYfnwfwkG6hP" width="196" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/a/3a5032a1e7cc38b1125587cab20749939928d519_2_196x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/3/a/3a5032a1e7cc38b1125587cab20749939928d519_2_294x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/3/a/3a5032a1e7cc38b1125587cab20749939928d519_2_392x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/a/3a5032a1e7cc38b1125587cab20749939928d519_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename"></span><span class="informations">1588×4034 281 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
Has this button been replaced by the <a href="https://github.community/faq#where-can-i-provide-feedback-about-the-github-support-community-3">community feedback form</a>?
<p>The next thing I planned to try to contact the moderators about my post was the “Flag” button described in the <a href="https://github.community/faq">FAQ</a>, but I couldn’t see it either.  But as I was in the process of researching what to do, apparently I met the reading time threshold for the Basic <a href="https://blog.discourse.org/2018/06/understanding-discourse-trust-levels/" rel="noopener nofollow ugc">trust level</a>, and then the “Flag” button automatically appeared. :slightly_smiling_face:  Can an administrator please update the FAQ to make it more obvious that if the user can’t see the button, they should try spending some more time reading?</p>
<p>Thanks.</p>

**Comment by mattmccutchen** [comment]
<p>FWIW: I just received a message, “A staff member reviewed your post in <em>Using one account for all your projects</em> and it is now visible.”  So it seems that the notification feature (the initial subject of this thread) has been implemented.</p>

**Comment by turaling** [comment]
<p>Sorry to bring this up here but, could one of the staff members review the <a href="https://github.community/t/multiple-auth-mechanisms-are-not-allowed-error-when-downloading-release-assets/215746/">post</a> I created two days ago, please?</p>
